webpack.playground.config.js

```diff
@@ -9,6 +10,14 @@ const PLAYGROUND = path.resolve(__dirname, 'playground');
 const DEFAULT_SERVER_PORT = 3000;
 const WIDGET_RC_FILE = '.widgetrc.js';
 
+if (!fs.existsSync(WIDGET_RC_FILE) && fs.existsSync('.widgetrc')) {
```

Comment:
we should log a message if there is no .widgetrc file at all?

Reply:
why so if there is no `.widgetrc` which implies either dev already migrate or they didn't create it at first place.